### PR TITLE
chore: remove unused features and endpoints

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: api/async/publish.ts [hunk 1]
```diff
@@ -209,26 +201,16 @@ async function publishGithub(
     await uploadVersionMetaJson(
       moduleName,
       version,
-      "meta.json",
       {
         uploaded_at: new Date().toISOString(),
-        directory_listing: directory.sort((a, b) =>
-          a.path.localeCompare(b.path, "en-US")
-        ),
```
Comment (lucacasonato) [inline api/async/publish.ts:217]: Pretty sure this is still used by people. I would like to continue uploading this file.

Comment (crowlKats) [reply api/async/publish.ts:217]: apiland should be the source of truth for that, as it does have a directory listing capability (although not flattened).
dont see the need to keep duplicate similar things around, especially when we dont need this one at all anymore

Comment (lucacasonato) [reply api/async/publish.ts:217]: This data is always available without requiring any database to be online. This is a good trait to have - there is no cost for us to keep this static information in S3.

Comment (crowlKats) [reply api/async/publish.ts:217]: > Also, apiland gets it's directory listing from this file :)

you are right... aight will revert that change

File: api/async/publish_test.ts [hunk 2]
```diff
@@ -238,10 +234,6 @@ Deno.test({
         },
         status: "success",
         message: "Published module.",
-        stats: {
-          total_files: 2,
-          total_size: 425,
-        },
```
Comment (lucacasonato) [inline api/async/publish_test.ts:244]: Does the frontend use these on `/status/<id>`?

Comment (crowlKats) [reply api/async/publish_test.ts:244]: no

File: utils/storage.ts [hunk 3]
```diff
@@ -41,20 +41,6 @@ export async function getMeta(
   return new Uint8Array(data);
 }
 
-export async function getVersionMetaJson(
```
Comment (lucacasonato) [inline utils/storage.ts:44]: Was this never used?

**Review by lucacasonato — APPROVED** [review]

File: utils/storage.ts (hunk 3 above)
Comment (crowlKats) [reply utils/storage.ts:44]: its something i forgot to remove in #304

**Pull request merged**